sufia.gemspec

```diff
@@ -31,6 +31,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'select2-rails', '~> 3.5.9'
   spec.add_dependency 'json-schema'
   spec.add_dependency 'oauth'
+  spec.add_dependency 'devise', '~> 4.2'
```

Comment:
I opened a related ticket:
https://github.com/projectblacklight/blacklight/issues/1428

I think that conditional is inapplicable.  Our version of rails didn't change, but version of devise did.  I would fix Blacklight to specify a SemVer of devise that didn't require such a janky workaround.